src/commands/build_status_to_pagerduty.yml

```diff
@@ -0,0 +1,41 @@
+description: |
+  Send build status to PagerDuty integration.
+
+parameters:
+  integration-key:
+    description: Integration key (a.k.a routing key) provided by PagerDuty.
+    type: string
+  when:
+    description: When to send the notification to PagerDuty. Possible values are on_fail and on_success
+    type: string
+    default: on_fail
+
+steps:
+  - run:
+      when: << parameters.when >>
```

Comment:
we might as well let the caller specify the command why add it specifically as a param